rubygem-foreman_hooks/rubygem-foreman_hooks.spec

```diff
@@ -74,6 +74,9 @@ GEMFILE
 mkdir -p %{buildroot}%{foreman_dir}/config/hooks
 ln -s %{gem_instdir} %{buildroot}%{foreman_dir}/%{gem_name}
 
+# debug script
+ln -s %{gem_instdir}/extra/foreman-debug.sh %{foreman_dir}/script/foreman-debug.d/50-foreman_hooks
```

Comment:
1. Parent directory doesn't seem to be created
1. Symlink would have to be in the buildroot, not in `/`